python/dgl/nn/pytorch/graph_transformer.py

```diff
@@ -219,3 +223,113 @@ def forward(self, ndata, attn_bias=None, attn_mask=None):
         attn = self.out_proj(attn.reshape(N, bsz, self.feat_size).transpose(0, 1))
 
         return attn
+
+
+class GraphTransformerLayer(nn.Module):
+    r"""Graph Transformer Layer with Dense Multi-Head Attention, as introduced
+    in `Do Transformers Really Perform Bad for Graph Representation?
+    <https://arxiv.org/pdf/2106.05234>`__

```

Comment:
Yes, it is the same as Graphormer, but other graph transformer models use almost the same structure except the dropout layers and we want to make this layer universal to them.

Reply:
This might need a bit further discussion. The existing DGL practice names modules based on the original model name, e.g. `GraphConv` and `SAGEConv` are similar to many GNN models. cc @jermainewang @BarclayII for discussion